apps/api/src/main/java/dk/treecreate/api/designs/Design.java

```diff
@@ -0,0 +1,112 @@
+package dk.treecreate.api.designs;
+
+import dk.treecreate.api.user.User;
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "designs")
+public class Design
+{
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(
+        name = "UUID",
+        strategy = "org.hibernate.id.UUIDGenerator",
+        parameters = {
+            @org.hibernate.annotations.Parameter(
+                name = "uuid_gen_strategy_class",
+                value = "org.hibernate.id.uuid.CustomVersionOneStrategy"
+            )
+        }
+    )
+    @Type(type = "uuid-char")
+    @Column(name = "design_id", updatable = false, nullable = false)
+    @ApiModelProperty(notes = "UUID of the design entity",
+        example = "c0a80121-7ac0-190b-817a-c08ab0a12345")
+    private UUID designId;
+
+    @ManyToOne
+    @ApiModelProperty(notes = "The user the design belongs to")
+    private User user;
+
+    @Column(name = "design_properties", nullable = false)
+    @ApiModelProperty(notes = "Design-specific properties, as a JSON string",
+        example = "{ title: Example design," +
```

Comment:
Example design should be in quotes

Reply:
Resolved in e65dbb6
It works without them but I have added them anyway